tests/unit/models/test_input.py

```diff
@@ -211,7 +211,7 @@ def test_invalid_package_paths(self, path: str, expect_error: str, tmp_path: Pat
             )
 
     def test_invalid_flags(self) -> None:
-        expect_error = r"Input should be 'cgo-disable', 'force-gomod-tidy', 'gomod-vendor' or 'gomod-vendor-check'"
+        expect_error = r"Input should be 'dev-package-managers', 'cgo-disable', 'force-gomod-tidy', 'gomod-vendor' or 'gomod-vendor-check'"
```

Comment:
This commit needs to be squashed into the previous one, otherwise the unit test suite fails with the previous commit.

Reply:
> This commit needs to be squashed into the previous one, otherwise the unit test suite fails with the previous commit.

[tangentially related]
I just realized that we've already broken bisactability (i.e. making the unit test suite pass after every single commit) on a few occasions. Despite that fact, I still very strictly stick by ^my comment on making the test suite pass after every commit. I wonder whether it were of any use (or simply worth it) to come up with mechanisms to prevent this from happening again going forward now that we know for sure we've broken that "contract" (IOW making sure every commit is properly tested), e.g. declaring cachi2 v1.0.0 stable with more guarantees?

Reply:
I strive to make every commit pass the test suite when I'm writing code, but I don't run the suite for every single one of them, specially when I'm doing code reviews. If there's any automation you know of we could employ, it might be worth it.

Reply:
> If there's any automation you know of we could employ, it might be worth it.

I'm not familiar with GH actions yet, but my first simplistic idea would be to spin up another container and just run (just with a single Python environment, no need for all of them) `git rebase main -x 'make test-unit'`. With a bit of tweaking, i.e. without a shell, it should fail the whole GH action at first commit that doesn't pass the test suite.